src/components/nfts/NFTCardGallery.scss

```
@@ -0,0 +1,91 @@
+#card-gallery-container {
+  width: 280px;
+  padding: 15px 15px 24px 15px;
+  background-color: #21383d;
+  border-radius: 5px;
+  box-sizing: border-box;
+  cursor: pointer;
+
+  p {
+    margin: 0;
+  }
+
+  header {
+    display: flex;
+    margin-bottom: 15px;
+    column-gap: 8px;
+    justify-content: center;
+
+    p {
+      font-size: 14px;
+      font-weight: 700;
+      color: #88a2b3;
```

Even if this color does not exist on [index.scss](https://github.com/CityOfZion/dora/blob/6167155c400e66960685c636bdb39ec37ae02ec5/src/index.css#L52) , you can use `--medium-grey`

```suggestion
      color: var(--medium-grey);
```